nvflare/fuel/hci/tools/admin.py

```diff
@@ -100,6 +100,8 @@ def main():
         ca_cert=ca_cert,
         client_cert=client_cert,
         client_key=client_key,
+        upload_dir=admin_config.get("upload_dir"),
+        download_dir=admin_config.get("download_dir"),
```

Comment:
One note here, it need not to be resolved in this PR.

I think we don't need these two folders, and we should remove https://github.com/NVIDIA/NVFlare/blob/dev-2.1/nvflare/fuel/hci/client/api.py#L130-L133

I think the API itself does not need to have "FileTransferModule" built in, what do you think?